src/components/NcActions/NcActions.vue

```diff
@@ -702,7 +704,7 @@ export default {
 		 */
 		boundariesElement: {
 			type: Element,
-			default: () => document.querySelector('body'),
+			default: () => document?.querySelector('body'),
```

Comment:
I guess it's not. If I remember correctly, I saw errors here when first working on the vue 3 migration, but it works without checking for the existence of `document` now. Changed it.

Reply:
Maybe for unit tests, but if you unit test a DOM component you probably want some kind of DOM environment -> There should always be the document.

--- 

Well while writing this: There is one situation when the document is undefined: SSR.
Not sure if we ever will use SSR but it will fail here, because document is only defined after (and in) the mounted hook.

Reply:
Maybe we fix / adjust it when we do SSR and leave it like this until then.